wolfcrypt/src/random.c

```diff
@@ -3913,6 +3913,17 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
 
     #define USE_TEST_GENSEED
 
+#elif defined(WOLFSSL_MAXQ108X) || defined(WOLFSSL_MAXQ1065)
```

Comment:
I see this code already in random.c above at line 3829... I think you can remove this

Reply:
Good call, thanks; Fixed!